src/core/util.ts

```diff
@@ -11,53 +11,53 @@ import ULog from "ulog";
  * @see Log
  */
 export enum LogLevel {
-  ERROR = 1,
-  WARN = 2,
-  INFO = 3,
-  LOG = 4,
-  DEBUG = 5
+    ERROR = 1,
```

Comment:
Our spacing is 4, as per prettier config...

Reply:
Will disable spacing as well

Reply:
I think we would want
```
{
    "indent": ["error", 2]
}
```

Reply:
I think the preferred way is to disable the rule (like what's happening now) and let prettier handle this.